0000-catalog.md

```diff
@@ -710,7 +716,8 @@ message CatalogProductSetStatusAction {
 
 Validation requirements:
 
-- If a catalog_product with catalog_product_id does not exist the transaction is
+- The catalog_product must be able to be updated or the transaction is invalid
```

Comment:
I'm not sure what this means. This should be enough information for someone to implement the smart contract.

Reply:
added more info 
> (i.e. meaning the status is not "DISCONTINUED")